TrueNASCORE/livestats.blade.php

```diff
@@ -5,6 +5,6 @@
     </li>
     <li>
         <span class="title">Alerts</span>
-        <strong>{!! $alert_tot !!}</strong>
+        <strong>{!! $alert_tot !!}  / <span style="background-color: #d64d55; padding: 3px; color: white; font-size: 12px; border-radius: 3px; opacity: 1;">{!! $alert_crit !!}</span></strong>
```

Comment:
I thought this color was supposed to by red? (based on your screenshot)

Reply:
Yes the background color is red, the text color is white.

Reply:
hahah sorry, missed that. doh. :smile: